SYCL/ESIMD/regression/bit_shift_compilation_test.cpp

```diff
@@ -178,16 +178,6 @@ int main(int argc, char *argv[]) {
       q, TEST_VALUE, 1, 0x7FFFFFFF, 0x2AAAAAAA, 0xAAAAAAAA, 0xAAAAAAAA,
       0x2AAAAAAA, 0x2AAAAAAA);
 
-  test_result |= test<uint16_t, uint16_t, int32_t>(q, 0x8000, 1, 0x0, 0x4000,

```

Comment:
I'd rather comment that section out, add a TODO comment and investigate it separately.